### DISCUSSION TITLE
Cursor jumping at window edges on Linux

### DISCUSSION THREAD

**Discussion by rXelelo** [ppy/osu#28903] [Q&A] [answered]
### Type

Game behaviour

### Bug description

wtf with cursor

### Screenshots or videos

[Запис з 2024-07-18 10-49-48.webm](https://github.com/user-attachments/assets/2a692542-d53d-42be-a410-4476d3989d15)

### Version

2024.718.0

### Logs

nothing

**Comment by smoogipoo** [comment]
Logs?

**Comment by rXelelo** [comment]
> Logs?
where  to find

**Comment by rXelelo** [comment]
> Logs?

[compressed-logs.zip](https://github.com/user-attachments/files/16279704/compressed-logs.zip)

**Comment by smoogipoo** [comment, marked as answer]
Try running with environment variable `OSU_SDL3=1`.

**Comment by rXelelo** [comment]
> OSU_SDL3=1

how?

**Reply by smoogipoo** [reply]
It depends on how you installed osu!, but from the terminal.

Something like
```sh
$ OSU_SDL3=1 ./osu\!.AppImage
```
or
```sh
$ export OSU_SDL3=1
$ ./osu\!.AppImage

**Comment by rXelelo** [comment]
thanks

**Reply by smoogipoo** [reply]
Did that fix it?
